components/Header/index.js

```diff
@@ -1,3 +1,21 @@
-import Header from './Header';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import DefaultHeader from './DefaultHeader';
+import TransparentHeader from './TransparentHeader';
+
+const Header = ({ type }) => {
+  switch (type) {
+    case 'transparent':
+      return <TransparentHeader />;
+    default:
+      return <DefaultHeader />;
+  }
+};
+
+Header.propTypes = {
+  type: PropTypes.oneOf(['transparent']),
```

Comment:
Let's also add `default` as an option in order to make things REALLY explicit for readability
```suggestion
  type: PropTypes.oneOf(['transparent', 'default']),
```